composer.json

```diff
@@ -5,6 +5,7 @@
     "homepage": "https://github.com/php-coveralls/php-coveralls",
     "type": "library",
     "license": "MIT",
+
```

Comment:
i believe we can drop the extra empty lines or use https://github.com/ergebnis/composer-normalize

Reply:
Ok. I will check it